tokens/config.js

```diff
@@ -6,11 +6,11 @@ module.exports = {
       buildPath: "build/css/",
       files: [
         {
-          destination: "_variables.css",
+          destination: "variables.css",
```

Comment:
Nit: I kinda like the `_` prefix as it hints that it's a partial to be included with other things. I'm not married to it, but just curious about your thoughts for dropping it.

Reply:
My thought was it's a bit weird for the main import for dependent projects to be prefixed with an _ which was seeming like a private internal file.